apm-agent-plugins/apm-grpc/apm-grpc-plugin/src/test/java/co/elastic/apm/agent/grpc/GrpcTest.java

```diff
@@ -26,10 +26,11 @@
 
 import co.elastic.apm.agent.grpc.testapp.GrpcApp;
 import co.elastic.apm.agent.grpc.testapp.GrpcAppProvider;
+import co.elastic.apm.agent.testutils.TestPort;
 
 public class GrpcTest {
 
-    private static final int PORT = 50051;
+    private static final int PORT = TestPort.getAvailableRandomPort();
```

Comment:
this allows to have more than one test running on the same host without conflict.